
Index: git/po/fr/docs/kphotoalbum/index.docbook
===================================================================
--- git.orig/po/fr/docs/kphotoalbum/index.docbook	2018-01-22 20:48:51.785370719 +0100
+++ git/po/fr/docs/kphotoalbum/index.docbook	2018-01-22 20:49:27.437123307 +0100
@@ -35,7 +35,11 @@
 >Pedersen</surname
 > </author>
       
-      &traducteurXavierBesnard; 
+      <othercredit role="translator">
+		  <firstname>Xavier</firstname>
+		  <surname>Besnard</surname>
+		  <affiliation><address><email></email></address></affiliation><contrib>Übersetzung</contrib>
+      </othercredit>
     </authorgroup>
     
     <copyright>
Index: git/po/nl/docs/kphotoalbum/index.docbook
===================================================================
--- git.orig/po/nl/docs/kphotoalbum/index.docbook	2018-01-22 20:48:54.049354874 +0100
+++ git/po/nl/docs/kphotoalbum/index.docbook	2018-01-22 20:49:29.797107087 +0100
@@ -42,7 +42,25 @@
 >Turkia</surname
 > </author>
       
-      &Freek.de.Kruijf;&Ronald.Stroethoff; 
+      <othercredit role="translator">
+		  <firstname>Ged</firstname>
+		  <surname>Vertaald</surname>
+<affiliation
+><address
+><email
+></email
+></address
+></affiliation
+><contrib
+>Übersetzung</contrib
+>
+	  </othercredit> 
+      <othercredit role="translator">
+		  <firstname>Freek.de</firstname>
+		  <surname>Kruijf</surname>
+		  <affiliation><address><email></email></address></affiliation><contrib>Übersetzung</contrib>
+      </othercredit>
+      &Ronald.Stroethoff;
     </authorgroup>
     
     <copyright>
