Index: git/doc/fr/index.docbook
===================================================================
--- git.orig/doc/fr/index.docbook	2014-07-20 20:01:07.806126135 +0200
+++ git/doc/fr/index.docbook	2014-07-20 20:01:07.802126172 +0200
@@ -34,8 +34,12 @@
 > <surname
 >Pedersen</surname
 > </author>
-      
-      &traducteurXavierBesnard; 
+
+      <othercredit role="translator">
+		  <firstname>Xavier</firstname>
+		  <surname>Besnard</surname>
+		  <affiliation><address><email></email></address></affiliation><contrib>Übersetzung</contrib>
+      </othercredit>
     </authorgroup>
     
     <copyright>
Index: git/doc/nl/index.docbook
===================================================================
--- git.orig/doc/nl/index.docbook	2014-07-20 20:01:07.806126135 +0200
+++ git/doc/nl/index.docbook	2014-07-20 20:02:30.669462331 +0200
@@ -42,7 +42,25 @@
 >Turkia</surname
 > </author>
       
-      &Freek.de.Kruijf;&Ronald.Stroethoff; 
+      <othercredit role="translator">
+		  <firstname>Ged</firstname>
+		  <surname>Vertaald</surname>
+<affiliation
+><address
+><email
+></email
+></address
+></affiliation
+><contrib
+>Übersetzung</contrib
+>
+	  </othercredit> 
+      <othercredit role="translator">
+		  <firstname>Freek.de</firstname>
+		  <surname>Kruijf</surname>
+		  <affiliation><address><email></email></address></affiliation><contrib>Übersetzung</contrib>
+      </othercredit>
+      &Ronald.Stroethoff;
     </authorgroup>
     
     <copyright>
