
Index: git/po/nl/docs/kphotoalbum/index.docbook
===================================================================
--- git.orig/po/nl/docs/kphotoalbum/index.docbook	2023-09-24 09:06:45.612518749 +0200
+++ git/po/nl/docs/kphotoalbum/index.docbook	2023-09-24 09:06:45.608518738 +0200
@@ -54,7 +54,25 @@
 >Turkia</surname
 > </author>
       
-      &Freek.de.Kruijf;&Ronald.Stroethoff; 
+      <othercredit role="translator">
+		  <firstname>Ged</firstname>
+		  <surname>Vertaald</surname>
+<affiliation
+><address
+><email
+></email
+></address
+></affiliation
+><contrib
+>Übersetzung</contrib
+>
+	  </othercredit> 
+      <othercredit role="translator">
+		  <firstname>Freek.de</firstname>
+		  <surname>Kruijf</surname>
+		  <affiliation><address><email></email></address></affiliation><contrib>Übersetzung</contrib>
+      </othercredit>
+      &Ronald.Stroethoff;
     </authorgroup>
     
     <copyright>
