Index: git/doc/fr/index.docbook
===================================================================
--- git.orig/doc/fr/index.docbook	2013-01-30 18:26:53.000000000 +0100
+++ git/doc/fr/index.docbook	2013-01-30 18:27:21.000000000 +0100
@@ -35,8 +35,12 @@
 > <surname
 >Pedersen</surname
 > </author>
-      
-      &traducteurXavierBesnard; 
+
+      <othercredit role="translator">
+		  <firstname>Xavier</firstname>
+		  <surname>Besnard</surname>
+		  <affiliation><address><email></email></address></affiliation><contrib>Übersetzung</contrib>
+      </othercredit>
     </authorgroup>
     
     <copyright>
Index: git/doc/nl/index.docbook
===================================================================
--- git.orig/doc/nl/index.docbook	2013-01-30 18:26:53.000000000 +0100
+++ git/doc/nl/index.docbook	2013-01-30 18:27:21.000000000 +0100
@@ -43,7 +43,24 @@
 >Turkia</surname
 > </author>
       
-      &ged.vertaald;&Freek.de.Kruijf; 
+      <othercredit role="translator">
+		  <firstname>Ged</firstname>
+		  <surname>Vertaald</surname>
+<affiliation
+><address
+><email
+></email
+></address
+></affiliation
+><contrib
+>Übersetzung</contrib
+>
+	  </othercredit> 
+      <othercredit role="translator">
+		  <firstname>Freek.de</firstname>
+		  <surname>Kruijf</surname>
+		  <affiliation><address><email></email></address></affiliation><contrib>Übersetzung</contrib>
+      </othercredit>
     </authorgroup>
     
     <copyright>
