Description: KDE svn stable branch pull
Author: Fathi Boudra <fabo@debian.org>
Bug-Debian: http://bugs.debian.org/563821
Forwarded: not-needed
Last-Update: 2010-01-14

--- a/AnnotationDialog/ShortCutManager.cpp
+++ b/AnnotationDialog/ShortCutManager.cpp
@@ -24,7 +24,7 @@ void AnnotationDialog::ShortCutManager::
         QWidget* widget = pair.second;
         QString title = dock->windowTitle();
         for ( int index = 0; index < title.length(); ++index ) {
-            const QChar ch = title[index];
+            const QChar ch = title[index].toLower();
             if ( !m_taken.contains(ch) ) {
                 m_taken.insert( ch );
                 dock->setWindowTitle( title.left(index) + QString::fromLatin1("&") + title.mid(index) );
@@ -37,7 +37,7 @@ void AnnotationDialog::ShortCutManager::
     Q_FOREACH( QLabel* label, m_labelWidgets ) {
         const QString title = label->text();
         for ( int index = 0; index < title.length(); ++index ) {
-            const QChar ch = title[index];
+            const QChar ch = title[index].toLower();
             if ( !m_taken.contains(ch) ) {
                 m_taken.insert( ch );
                 label->setText( title.left(index) + QString::fromLatin1("&") + title.mid(index) );
@@ -58,5 +58,5 @@ void AnnotationDialog::ShortCutManager::
 {
     const int index = text.indexOf(QChar::fromLatin1('&'));
     if ( index != -1 )
-        m_taken.insert( text[index+1] );
+        m_taken.insert( text[index+1].toLower() );
 }
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,7 +1,31 @@
+2010-01-06  Miika Turkia <theBro@luukku.com>
+
+	* Remove of some code that became dead with yesterdays bugfix
+
+2010-01-05  Miika Turkia <theBro@luukku.com>
+
+	* fixed bug #220373: Viewer settings dialog did not load correct
+	values and had wrong defaults
+
+2009-12-30  Jesper K. Pedersen  <blackie@kdab.com>
+	* fixed bug #220376: Shortcuts in the annotation dialog was broken
+	in some translations
+	* bugfix: KPA crashes when viewing images if viewer cache size is set
+	to zero
+
+2009-11-25 Hassan Ibraheem <hasan.ibraheem@gmail.com>
+
+	* fixed bug #211907: Fix EXIF support dialog at startup. Thanks to Olivier Berger
+
+2009-11-19 Hassan Ibraheem <hasan.ibraheem@gmail.com>
+
+	* fixed bug #147891: Crashing on wrong export directory in Plugin interface
+
 2009-11-08  Hassan Ibraheem <hasan.ibraheem@gmail.com>
 
 	* fixed bug #211392: Incorrect rotation angle after annotating multiple
 	 images.Thanks to Andreas Neustifter.
+
 ====================== KPhotoalbum 4.1 released  ===================================	
 
 2009-09-02  Miika Turkia <theBro@luukku.com>
--- a/DB/FileInfo.cpp
+++ b/DB/FileInfo.cpp
@@ -53,7 +53,7 @@ bool DB::FileInfo::updateDatFromFileTime
         return false;
 
     // If we are not setting date, then we should of course not set the date
-    if ( (mode & EXIFMODE_DATE) != 0 )
+    if ( (mode & EXIFMODE_DATE) == 0 )
         return false;
 
     // If we are we already have specifies that we want to sent the date (from the ReReadExif dialog), then we of course should.
--- a/Exif/SearchDialog.cpp
+++ b/Exif/SearchDialog.cpp
@@ -55,20 +55,24 @@ Exif::SearchDialog::SearchDialog( QWidge
     makeISO( grid );
     makeExposureTime( grid );
 
-    Q3Grid* grid2 = new Q3Grid( 4, settings );
-    grid2->setSpacing( 6 );
-    hlay->addWidget( grid2 );
+    grid = new Q3Grid( 4, settings );
+    grid->setSpacing( 6 );
+    hlay->addWidget( grid );
     hlay->addStretch( 1 );
-    _apertureValue = makeApertureOrFNumber( i18n( "Aperture Value" ), QString::fromLatin1( "Exif_Photo_ApertureValue" ), grid2 );
-    _fNumber = makeApertureOrFNumber( i18n( "F Number" ), QString::fromLatin1( "Exif_Photo_FNumber" ), grid2 );
+    _apertureValue = makeApertureOrFNumber( i18n( "Aperture Value" ), QString::fromLatin1( "Exif_Photo_ApertureValue" ), grid );
+    _fNumber = makeApertureOrFNumber( i18n( "F Number" ), QString::fromLatin1( "Exif_Photo_FNumber" ), grid );
 
     // Focal length
+    grid = new Q3Grid( 4, settings );
+    grid->setSpacing( 6 );
+    hlay->addWidget( grid );
+    hlay->addStretch( 1 );
     QLabel* label = new QLabel( i18n( "Focal Length" ), grid );
-    _fromFocalLength = new QSpinBox;
+    _fromFocalLength = new QSpinBox( grid );
     _fromFocalLength->setRange( 0, 10000 );
     _fromFocalLength->setSingleStep( 10 );
     label = new QLabel( i18n("to"), grid );
-    _toFocalLength = new QSpinBox;
+    _toFocalLength = new QSpinBox( grid );
     _toFocalLength->setRange( 0, 10000 );
     _toFocalLength->setSingleStep( 10 );
 
--- a/main.cpp
+++ b/main.cpp
@@ -39,8 +39,8 @@ int main( int argc, char** argv ) {
                           ki18n("KDE Photo Album"), KAboutData::License_GPL,
                           KLocalizedString(), KLocalizedString(), "http://www.kphotoalbum.org");
     aboutData.addAuthor( ki18n("Jesper K. Pedersen"), ki18n("Development"), "blackie@kde.org" );
-    aboutData.addAuthor( ki18n("Hassan Ibraheem"),ki18n("Development"), "hasan.ibraheem@gmail.com>");
-    aboutData.addAuthor( ki18n("Miika Turkia"),ki18n("Development"), "theBro@luukku.com>");
+    aboutData.addAuthor( ki18n("Hassan Ibraheem"),ki18n("Development"), "hasan.ibraheem@gmail.com");
+    aboutData.addAuthor( ki18n("Miika Turkia"),ki18n("Development"), "theBro@luukku.com");
     aboutData.addAuthor( ki18n("Tuomas Suutari"), ki18n("SQL backend and numerous features"), "thsuut@utu.fi" );
     aboutData.addAuthor( ki18n("Jan Kundrat"),ki18n("Development"), "jkt@gentoo.org");
     aboutData.addAuthor( ki18n("Henner Zeller"),ki18n("Development"), "h.zeller@acm.org");
--- a/Plugins/ImageInfo.cpp
+++ b/Plugins/ImageInfo.cpp
@@ -106,9 +106,11 @@ void Plugins::ImageInfo::setDescription(
 
 void Plugins::ImageInfo::clearAttributes()
 {
+    if( _info ) {
     _info->clearAllCategoryInfo();
     // TODO I think it is reasonable to keep the gps position anyway, isn't it?
     MainWindow::DirtyIndicator::markDirty();
+    }
 }
 
 void Plugins::ImageInfo::addAttributes( const QMap<QString,QVariant>& map )
--- a/Settings/SettingsDialog.cpp
+++ b/Settings/SettingsDialog.cpp
@@ -105,7 +105,7 @@ void Settings::SettingsDialog::show()
     _thumbnailsPage->loadSettings(opt);
     _subCategoriesPage->loadSettings();
     _databaseBackendPage->loadSettings(opt);
-    _viewerPage->reset(opt);
+    _viewerPage->loadSettings(opt);
 
 #ifdef HASKIPI
     _pluginsPage->loadSettings(opt);
--- a/Settings/ViewerPage.cpp
+++ b/Settings/ViewerPage.cpp
@@ -91,9 +91,3 @@ void Settings::ViewerPage::saveSettings(
     opt->setSlideShowSize( _slideShowSetup->size() );
     opt->setLaunchSlideShowFullScreen( _slideShowSetup->launchFullScreen() );
 }
-
-void Settings::ViewerPage::reset(Settings::SettingsData* opt )
-{
-    _viewImageSetup->setLaunchFullScreen( opt->launchViewerFullScreen() );
-    _slideShowSetup->setLaunchFullScreen( opt->launchSlideShowFullScreen() );
-}
--- a/Settings/ViewerPage.h
+++ b/Settings/ViewerPage.h
@@ -16,7 +16,6 @@ public:
     ViewerPage( QWidget* parent );
     void loadSettings( Settings::SettingsData* );
     void saveSettings( Settings::SettingsData* );
-    void reset(Settings::SettingsData* opt );
 
 private:
     Settings::ViewerSizeConfig* _slideShowSetup;
--- a/Viewer/ImageDisplay.cpp
+++ b/Viewer/ImageDisplay.cpp
@@ -417,7 +417,7 @@ void Viewer::ImageDisplay::updatePreload
     int incr = ( _forward ? 1 : -1 );
     int nextOnesInCache = 0;
     // Iterate from the current image in the direction of the viewing
-    for ( int i = _curIndex+incr; true ; i += incr ) {
+    for ( int i = _curIndex+incr; cacheSize ; i += incr ) {
         if ( _forward ? ( i >= (int) _imageList.count() ) : (i < 0) )
             break;
 
--- a/XMLDB/FileReader.cpp
+++ b/XMLDB/FileReader.cpp
@@ -259,7 +259,7 @@ void XMLDB::FileReader::checkIfImagesAre
 
     if ( wrongOrder ) {
         KMessageBox::information( messageParent(),
-#ifdef HASEXIV2
+#ifdef HAVE_EXIV2
                                   i18n("<p>Your images/videos are not sorted, which means that navigating using the date bar "
                                        "will only work suboptimally.</p>"
                                        "<p>In the <b>Maintenance</b> menu, you can find <b>Display Images with Incomplete Dates</b> "
