From d83730a934437f18e0406bddb244a7a9c497565c Mon Sep 17 00:00:00 2001
From: Johannes Zarl <johannes@zarl.at>
Date: Thu, 28 May 2015 19:49:52 +0200
Subject: [PATCH] Fix compilation for Ubuntu 15.04.

---
 Plugins/ImageInfo.cpp   | 2 +-
 Settings/BirthdayPage.h | 1 +
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/Settings/BirthdayPage.h b/Settings/BirthdayPage.h
index e61d2fe..2fb5325 100644
--- a/Settings/BirthdayPage.h
+++ b/Settings/BirthdayPage.h
@@ -30,6 +30,7 @@ class QFont;
 class QCalendarWidget;
 class QPushButton;
 class QTableWidgetItem;
+class QLabel;
 
 // KDE classes
 class KPageWidgetItem;
-- 
2.6.4

